<launch>
  <!--- Run AMCL -->
  <!-- include file="$(find cwru_nav)/amcl_diff.launch" /-->

  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    <rosparam file="$(find cwru_configs)/$(optenv ROBOT sim)/nav/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find cwru_configs)/$(optenv ROBOT sim)/nav/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find cwru_configs)/$(optenv ROBOT sim)/nav/costmap_local_params.yaml" command="load" />
    <rosparam file="$(find cwru_configs)/$(optenv ROBOT sim)/nav/costmap_global_params_outdoor.yaml" command="load" />
    <rosparam file="$(find cwru_configs)/$(optenv ROBOT sim)/nav/base_local_planner_params_outdoor.yaml" command="load" />
  </node>

  <node pkg="cwru_nav" type="bounding_box_generator.py" name="bounding_box_generator">
    <param name="box_height" value="65.0" />
    <param name="box_width" value="55.0" />
    <param name="step_size" value="0.1" />
    <param name="frame_id" value="bounding_box" />
    <param name="publishing_rate" value="20" />
  </node>


  <node pkg="tf" type="static_transform_publisher" name="bounding_box_transform_publisher" args="123.42 -25.62 1.9 0.0 0.0 0.0 odom bounding_box 100" />
</launch>
